paddle/phi/api/yaml/op_compat.yaml

```diff
@@ -1393,6 +1393,12 @@
   extra :
     attrs : [bool use_mkldnn = false]
 
+- op : nextafter
+  inputs :
+    {x: X, y : Y}
+  outputs :
+    out: Out
+
```

Comment:
No need to add `nextafter` in `paddle/phi/api/yaml/op_compat.yaml`, because there is a comment at the top of this file stating: There's no need to configure this file for new operator. 
The operator in this file is for historical compatibility reasons and will be gradually removed after standardization in the future.